improver/blending/spatial_weights.py

```diff
@@ -317,7 +317,7 @@ def process(self, cube_to_collapse, one_dimensional_weights_cube, blend_coord):
 
         Returns:
             iris.cube.Cube:
-                A cube containing normalised spatial weights based on the
+                A cube containing non-normalised spatial weights based on the
```

Comment:
Sorry I think you need to check the class docstring too.

Reply:
Ah, apologies, I see it, I'll fix it now